demo/site/src/documentTypes/Page.tsx

```diff
@@ -43,10 +41,7 @@ const pageQuery = gql`
 `;
 
 export default async function Page({ pageTreeNodeId, scope }: { pageTreeNodeId: string; scope: GQLPageTreeNodeScopeInput }) {
-    let previewData: SitePreviewData | undefined = undefined;
-    if (draftMode().isEnabled) {
-        previewData = { includeInvisible: false };
-    }
+    const { previewData } = previewParams() || { previewData: undefined };
```

Comment:
then this would be more complicated:
```
const { scope, previewData } = previewParams() || { scope: { domain, language: defaultLanguage }, previewData: undefined };
```